src/agenda/reservation-list/reservation.tsx

```diff
@@ -75,7 +75,7 @@ class Reservation extends Component<ReservationProps> {
   }
 
   renderDate(date?: XDate, item?: DayReservations) {
-    if (isFunction(this.props.renderDay) && date) {
+    if (isFunction(this.props.renderDay)) {
       return this.props.renderDay(date, item);
```

Comment:
You need to change the type on 'renderDay' prop. date param must be optional

Reply:
That's the ts error you get: `src/agenda/reservation-list/reservation.tsx(79,35): error TS2345: Argument of type 'XDate | undefined' is not assignable to parameter of type 'XDate'.
14:56:22   Type 'undefined' is not assignable to type 'XDate'.`